pinc/access_change_emails.inc

```diff
@@ -56,6 +56,95 @@ function pp_access_change_email($user, $access_change)
+
+    $message[] = _("If you remember how your mentor guided you when you started, you may consider giving gentle feedback in a private message to new proofreaders when you see them making repeated errors.");
+
+    $message[] = _("The text you worked on in P1 was generated from the page images by means of software tools. When you work in P2, however, you are working on text that someone else has already proofread. Nevertheless, please check every character and word carefully, since new P1s, unfamiliar with our guidelines, may have left errors.");
+
+    if ($daily_limit) {
+        $message[] = sprintf(_("Please keep in mind that we currently have a limit in P2 of no more than %d pages per day. This is a daily limit, not an average; the limit is there to make sure that we have a variety of projects available in the round at all times."), $daily_limit);
+        // The above paragraph will be added to the email only if the daily limit
+        // is a positive integer. Currently, if the daily limit value is zero or
+        // NULL, the paragraph is not added. This code will need to be adjusted
+        // if there is ever a need for them to be treated differently.
+    }
+
+    $message[] = "== " . _("What resources are available?") . " ==";
+
+    $message[] = "- " . sprintf(_("You may ask for help at dp-feedback ( %s ), giving the name of the project, the round you worked in, and the png numbers of the pages you would like to have reviewed."), "https://www.pgdp.net/phpBB3/ucp.php?i=pm&mode=compose&u=37603");
+
+    $message[] = "- " . sprintf(_("If you haven't yet, try using DP’s Proofreading Font: %s"), "https://www.pgdp.net/c/faq/font_sample.php");
+
+    $message[] = "- " . sprintf(_("When the diffs for your pages become available, check your diffs: %s"), "https://www.pgdp.net/wiki/Checking_your_diffs");
+
+    $message[] = "- " . sprintf(_("For a quick refresher, consider redoing the proofreading quizzes: %s"), "https://www.pgdp.net/c/quiz/start.php?show_only=PQ");
+
+    $message[] = _("If you have questions, there are several options:");
+
+    $message[] = "- " . _("For questions specific to a project you're working on, please post your question in the project's Project Discussion. There's a link to this on each project's main Project Page on the line reading \"Forum\".");
+
+    $message[] = "- " . sprintf(_("If you have a general question, you can post in the \"Common Proofreading Q&A\" Forum: %s"),
+        "https://www.pgdp.net/phpBB3/viewforum.php?f=19");
+
+    $message[] = "- " . _("If you don't know whether the question is a general question or specific to the project, you should start by asking in the Project Discussion.");
+
+    $message[] = "- " . _("If you'd rather ask a question privately or would like more information on proofreading in general, you can send a Private Message to one of the following volunteers:") .
+        "\n    Bess (Bess): https://www.pgdp.net/phpBB3/ucp.php?i=pm&mode=compose&u=101542" .
```

Comment:
Do we want maybe to point to a wiki page to make keeping the list of people who want to be asked up to date easier?

Reply:
I'm happy to have it in the code -- we don't change that that much and are less likely to forget to change it in the letter than in the wiki.